Code/Framework/AzCore/AzCore/Utils/Utils.h

```diff
@@ -104,6 +104,7 @@ namespace AZ
         // Attempts the supplied path to an absolute path.
         //! Returns nullopt if path cannot be converted to an absolute path
         AZStd::optional<AZ::IO::FixedMaxPathString> ConvertToAbsolutePath(AZStd::string_view path);
+        bool ConvertToAbsolutePath(AZ::IO::FixedMaxPath& outputPath, AZStd::string_view path);
         bool ConvertToAbsolutePath(const char* path, char* absolutePath, AZ::u64 absolutePathMaxSize);
```

Comment:
Suggestion: seems like the new version of ConvertToAbsolutePath should have (path, outputAbsolutePath) as inputs instead of the reverse, so that the ordering lines up with the other version.

Reply:
+1

Reply:
Well the order was made to be consistent with the file IO functions(ResolvePath, ResolveAlias, ConvertToAlias), where the output FixedMaxPath is the first parameter: https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/IO/FileIO.h#L250-L256

Reply:
Another option here then would be to change the second ConvertToAbsolutePath() to push the const char* path to the end. It just seems a little error-prone to have two versions of the same method take inputs and outputs in opposite order from each other.